Add Web3 University to online communities and support pages

btandowsky opened the pull request:
Adding Web3 University to the developer bootcamp page. I did this by 

1. Adding Web3 University to learning-tools.js 
2. Uploading a Web3 university image
3. Adding a description. 

Please let me know if there are any issues to the code! I am excited to contribute!

wackerow commented on src/pages/developers/learning-tools.js:
```diff
@@ -360,6 +369,9 @@ export const query = graphql`
     }
     nftschool: file(relativePath: { eq: "dev-tools/nftschool.png" }) {
       ...learningToolImage
+    }
+        web3university: file(relativePath: { eq: "dev-tools/web3university.png" }) {
+      ...learningToolImage
```
```suggestion
    }
    web3university: file(relativePath: { eq: "dev-tools/web3university.png" }) {
      ...learningToolImage
```

wackerow reviewed (approved):
@btandowsky Thanks for posting =) Looks good to me!

btandowsky commented:
> Hey @btandowsky thanks for the suggestion here 🙏
> 
> IMO this doesn't really fit into the developer bootcamp bucket. This seems like a curation/collection of valuable resources but I wouldn't say it has the kind of structured learning path we'd want to list this as a bootcamp. Definitely agree we should link out to this valuable resource _somewhere_ just not entirely sure where the best fit is. Our [/learn/](https://ethereum.org/en/learn/) page is an obvious one but I think we can do better. Any thoughts?


Thanks for getting back to me back to me @minimalsm . I am open to placing it in learn community guide and resources page under "In addition to the information on this page, there are many community-built resources worth exploring:" I would also like to add our awesome [discord](https://discord.gg/ZH5aXDgWEU) to the Online communities tab under chat rooms? What do you think?

minimalsm commented:
Yeah @btandowsky I think adding it as an online chatroom makes 💯. Feel free to add these changes to this PR, or open a new PR if you feel that is easier.

btandowsky commented:
> Yeah @btandowsky I think adding it as an online chatroom makes sense 💯. Feel free to add these changes to this PR, or open a new PR if you feel that is easier.

@minimalsm just updated, please let me know if this works! Summary of what I added.

1. Added Web3 University discord to online communities and Web3 University website building dAPPs sections under community
2. Added Alchemy discord to tooling.

btandowsky commented:
Hi team I wanted to check in and see if there were any questions with the updates? Please let me know!

minimalsm commented on src/intl/en/page-developers-learning-tools.json:
```diff
@@ -37,5 +37,7 @@
-  "page-learning-tools-nftschool-logo-alt": "NFT school Logo"
+  "page-learning-tools-nftschool-logo-alt": "NFT school Logo",
+  "page-learning-tools-web3university-description": "Covering the fundamentals of web3 development: from writing Solidity, to minting NFTs, to building full-stack dApps.",
+  "page-learning-tools-web3university-logo-alt": "Web3 University logo"  
```
```suggestion
```

minimalsm commented on src/content/learn/index.md:
```diff
@@ -27,6 +27,7 @@ In addition to the information on this page, there are many community-built reso
 - [What’s new in ETH 2.0](https://eth2.news) _A regular newsletter about Eth2 development_
 - [ethresear.ch forum](https://ethresear.ch/) _Deeper technical discussions on Ethereum for Eth2 and beyond_
 - [ETHGlobal](https://ethglobal.co) _An Ethereum hackathon series - attend one near you!_
+- [Web3 University](https://www.web3.university) _A community driven online resource foused on teaching and learning Web3 development_
```
- Few typos/conventions
- Removed 'teaching and learning' as the sentence target was changing mid-sentence

```suggestion
- [Web3 University](https://www.web3.university) _A community-driven online resource focused on web3 development_
```

minimalsm commented on src/content/community/online/index.md:
```diff
@@ -28,6 +28,7 @@ Hundreds of thousands of Ethereum enthusiasts gather in these online forums to s
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/5W5tVb3">CryptoDevs</Link> <i>- Ethereum development focused Discord community</i></SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.io/ethstaker">EthStaker Discord</Link> <i>- community oriented around offering project management support to Ethereum development</i></SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/CetY6Y4">Ethereum.org website team</Link> <i>- stop by and chat ethereum.org web development and design with the team and folks from the community</i></SocialListItem>
+<SocialListItem socialIcon="discord"><Link to="https://discord.gg/ZH5aXDgWEU">web3.university website team</Link> <i>- community focused on learning Web3 development </i></SocialListItem>
```
```suggestion
<SocialListItem socialIcon="discord"><Link to="https://discord.gg/ZH5aXDgWEU">Web3 University</Link> <i>- community focused on learning web3 development </i></SocialListItem>
```

minimalsm reviewed (changes requested):
Hey @btandowsky, thanks for this 🙏

Just a few small changes and we can get this merged. Could you also remove the web3university.png image you added? As this is not a learning-tool the logo isn't required.

btandowsky commented:
@minimalsm Thanks for all of the suggestions! Made all the commit suggestions and LFTM now! Please let me know if anything else is needed before merge!

minimalsm commented on src/intl/en/page-developers-learning-tools.json:
```diff
@@ -37,5 +37,5 @@
   "page-learning-tools-vyperfun-logo-alt": "Vyper.fun logo",
   "page-learning-tools-nftschool-description": "Explore what's going on with non-fungible tokens, or NFTs from the technical side.",
-  "page-learning-tools-nftschool-logo-alt": "NFT school Logo"
+  "page-learning-tools-nftschool-logo-alt": "NFT school Logo",
```
```suggestion
```

minimalsm commented:
Thanks @btandowsky. Could you also remove the web3university.png image you added please 🙏?

btandowsky commented:
@minimalsm Thanks for all of the tips, just removed the final image by committing your suggestion. Are we good to merge?

btandowsky commented:
@minimalsm Just wanted to check and see if there were any blockers on my end before merge?

minimalsm commented on src/intl/en/page-developers-learning-tools.json:
```diff
@@ -37,5 +37,4 @@
   "page-learning-tools-vyperfun-description": "Learn Vyper building your own Pokémon game.",
   "page-learning-tools-vyperfun-logo-alt": "Vyper.fun logo",
   "page-learning-tools-nftschool-description": "Explore what's going on with non-fungible tokens, or NFTs from the technical side.",
```
```suggestion
  "page-learning-tools-nftschool-description": "Explore what's going on with non-fungible tokens, or NFTs from the technical side.",
  "page-learning-tools-nftschool-logo-alt": "NFT school Logo"
```
